src/main/resources/tags/smelters_axe_blocks.json

```diff
@@ -0,0 +1,40 @@
+{
+  "values": [
+    "minecraft:oak_log",
+    "minecraft:spruce_log",
+    "minecraft:birch_log",
+    "minecraft:jungle_log",
+    "minecraft:acacia_log",
+    "minecraft:dark_oak_log",
+    "minecraft:stripped_oak_log",
+    "minecraft:stripped_spruce_log",
+    "minecraft:stripped_birch_log",
+    "minecraft:stripped_jungle_log",
+    "minecraft:stripped_acacia_log",
+    "minecraft:stripped_dark_oak_log",
+    {
+      "id" : "minecraft:mangrove_log",
```

Comment:
Very small thing, but put mangrove at the end, so things are roughly in the order they were introduced into Minecraft

Reply:
okay